test/e2e/mysql/01-connect.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: connect-mysql
+  namespace: default
+spec:
+  backoffLimit: 5
+  template:
+    metadata:
+      labels:
+        e2e-test: mysql
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: connect
+          image: docker.io/bitnami/mysql:latest
+          imagePullPolicy: IfNotPresent
+          command:
+            - bash
+          args:
+            - -c
+            - cd /opt/bitnami/mysql
+            - mkdir data
+            - mysqld start
+            - echo "Testing Select...\n" && MYSQL_DATABASE="$MYSQL_DB" MYSQL_USER="$MYSQL_USER" MYSQL_PASSWORD="$MYSQL_PASSWORD" MYSQL_HOST="$MYSQL_HOST" MYSQL_PORT="$MYSQL_PORT" mysql -c "select 1;"
```

Comment:
trying this I get this error in the pod:
```
Testing Select...\n
ERROR 2003 (HY000): Can't connect to MySQL server on 'e2e-test-mysql-exoscale-8f119768-1027-4b9c-bca7-4c14f9ff9928.aivencloud.com:3306' (111)
```
@ccremer do you know if I'm missing something else?

Reply:
maybe it has to do with the ssl mode, I'm checking it now

Reply:
sorry, but I have no idea how to make it work without running the mysql daemon. Any suggestion?

Reply:
Is there no debug log or verbose option available in the client that could maybe give more insights?

Reply:
could it be that the passed env variables are not the correct ones?
e.g. https://dev.mysql.com/doc/refman/8.0/en/environment-variables.html lists `MYSQL_PWD` as the password env variable, though it also lists it as extremely insecure and deprecated. That it's insecure is clear, but because it's deprecated we may want to use a better approach:
https://dev.mysql.com/doc/refman/8.0/en/connecting.html
https://dev.mysql.com/doc/refman/8.0/en/option-files.html

Reply:
@ccremer @mweibel thanks for the hints. 
Main issue was that in mysql you need to use `-e` to execute a command from shell